modules/phase_field/test/tests/GrandPotentialPFM/GrandPotentialPFM.i

```diff
@@ -106,6 +106,24 @@
     type = BarrierFunctionMaterial
     eta = eta
   [../]
+  [./cs]
+    type = DerivativeParsedMaterial
+    args = 'w'
+    f_name = cs
+    material_property_names = 'A cseq'
+    function = 'w/A + cseq' # since w = A*(c-cseq)
+    derivative_order = 2
+    enable_jit = false
```

Comment:
I would remove this, unless you have a reason to want to disable just-in-time compilation. The default value is true. Enabling just-in-time compilation will generally improve convergence at a very low penalty in start up time. Let me know if you want to discuss this system further.

Reply:
removed in next commit [c1f81df](https://github.com/idaholab/moose/pull/16343/commits/c1f81df6db659c9fc4367a80036eaab5f84514d5)